lib/arduino-upload.coffee

```diff
@@ -187,6 +210,9 @@ module.exports = ArduinoUpload =
 			}
 			callback code, info
 			output.finish()
+			if atom.config.get('arduino-upload.autosave') == 'save+reopen'
```

Comment:
`if atom.config.get('arduino-upload.autosave') == 'save+reopen' and serialWasOpen` why split into two if-conditions?

Reply:
also this could use the `autosaveConf` variable if you use that up there